src/app/organizations/manage-organizations/manage-organizations.component.scss

```diff
@@ -58,7 +58,8 @@
       border-radius: 1px;
     }
     &__img img {
-      object-fit: fill;
+      object-fit: cover;
+      filter: blur(0.5px);
```

Comment:
why this? just asking.

Reply:
I understand the intent, but could you please try to upload a lower quality photo and see what will happen when putting a blur on it.

Reply:
Right, I understand what you mean. We can't be sure that users will provide high res images all the time. I'm going to remove this.